docs/sdks/uid2-sdk-ref-python.md

```diff
@@ -305,6 +312,20 @@ else:
 
 For a full example, see the `sample_sharing_client.py` in [examples/sample_sharing_client.py](https://github.com/IABTechLab/uid2-client-python/blob/main/examples/sample_sharing_client.py).
 
+## Developement
+
+### Example Usage
+You can run specific examples from the [examples](https://github.com/IABTechLab/uid2-client-python/blob/main/examples) directory
+
+```py
+python3 examples/sample_bidstream_client.py $BASE_URL $AUTH_KEY $SECRET_KEY $DOMAIN_NAME $AD_TOKEN
+```
+
+### Running tests
```

Comment:
We should have some brief explanatory copy below the heading and before the code example.